Core/TransactionsImporter.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.Collections.Generic;
+using System.IO;
+
+namespace Core
+{
+	public abstract class TransactionsImporter
+	{
+		public bool CanLoad(Stream file)
+		{
+			try
+			{
+				Load(file);
+			}
+			catch
```

Comment:
Codacy found an issue: [Catch a list of specific exception subtype or use exception filters instead.](https://app.codacy.com/manual/siriak/MoneyManager/pullRequest?prid=5836065)